2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2021 @@
+overall (Vereinigung der Kanzlerinnen und Kanzler der Universitäten
+Deutschlands 2019), we specifically focus on the needs stemming from the
+development and maintenance of research software here.
+
+As already mentioned, research software development not only requires
+domain expertise, but also software development **education, skills, and
+competencies**. Currently, most of the domain researchers developing and
+maintaining domain-specific software technology never received
+professional training on software development (Wilson et al. 2014;
+Philippe et al. 2019). To enhance the productivity and sustainability of
+computer-based research, it is essential to integrate software
+development training into the education of domain researchers.
+
+Currently, a significant portion of the existing research software is
+developed by individuals or in small groups, primarily to serve their
+own requirements. This situation is unsatisfying in terms of
+collaboration and inefficient in terms of several groups spending
+resources on generating similar or even the same functionality. To
+enable and promote synergies, it is important to allocate resources for
+research software development and to build **communities**, as described
+in (Katz et al. 2019).
+
+Challenges
+----------
+
+We are currently facing a **lack of awareness** for the importance of
+research software as discussed in the section on p. . Moreover, there is
+**little recognition** for the efforts put into software development and
+maintenance. In consequence, software development in academic settings
+is mostly considered as a means to an end and sustainability is often
+not considered in project planning and grant proposals and contributes
+little to progressing research careers (Science Guide 2019; “San
+Francisco Declaration on Research Assessment” 2012). The main challenge
+here is the continued use of metrics that primarily leverage
+traditionally published articles and article citation numbers.
+
+In academia, developers of research software are typically domain
+researchers, and in particular if new areas are explored, the software
+development process itself has research character. Obviously, developing
+research software requires not only domain knowledge but also software
+development skills, and the researchers leading the software development
+process are often domain experts with substantial software development
+experience. However, the current academic system in Germany does not
+provide a defined **RSE role**. Limited-term positions are, at least
+currently within the main German academic system, often effectively the
+end of their career path, sometimes even a dead end. The challenge here
+is the lack of available permanent positions within the non-professorial
+academic faculty (“Mittelbau”) in Germany, as well as the already
+mentioned lack of recognition for efforts concerning research software
+for faculty appointments within domain sciences.
```

Comment:
This leads to ignorance in HR (human resources) in regards to RSE career advice/development. This also leads to non offering of training. No careers need no training.

Reply:
Stimmt, haben wir aber glaube auch schon.